src/station_cmd.cpp

```diff
@@ -4014,68 +4014,68 @@ const StationList *StationFinder::GetStations()
 	return &this->stations;
 }
 
+static bool CanMoveGoodsToStation(const Station *st, CargoID type) {
+	if (st->owner != OWNER_NONE && st->town->exclusive_counter > 0 && st->town->exclusivity != st->owner) return false;
```

Comment:
Missing comment on this line.
Also, with this being its own function now I'd suggest moving the end-of-line comments to the line before each check, to improve readability.